roboflow/config.py

```diff
@@ -1,26 +1,58 @@
+import json
 import os
 
+# default configuration location
+conf_location = os.getenv(
+    "ROBOFLOW_CONFIG_DIR", default=os.getenv("HOME") + "/.config/roboflow/config.json"
+)
+
+# read config file for roboflow if logged in from python or CLI
+if os.path.exists(conf_location):
+    with open(conf_location) as f:
+        config = json.load(f)
+else:
+    config = {}
+
+
+# read configuration variables conditionally
+
+
+##1. check if variable is in environment
+##2. check if variable is in config file
+##3. return default value
+def get_var_conditional(key, default):
+    if os.getenv(key) != None:
+        return os.getenv(key)
+    elif key in config.keys():
+        return config[key]
+    else:
+        return default
```

Comment:
Please, add a comment or use a better variables name to make it easier.

For instance, what's the type of `default`?

Reply:
for instance, I am not sure but the above code may be rewritten by

`os.environ.get(key, config.get(key, default))`

Reply:
It may indeed be written, but I like the cascade - going to keep it with the new comments